imagetest/test_suites/disk/disk_resize_test.go

```diff
@@ -32,19 +35,28 @@ func TestMain(m *testing.M) {
 
 // TestDiskResize Validate the filesystem is resized on reboot after a disk resize.
 func TestDiskResize(t *testing.T) {
-	_, err := os.Stat(markerFile)
+	image, err := utils.GetMetadata("image")
+	if err != nil {
+		t.Fatalf("couldn't get image from metadata")
+	}
+	switch {
+	case strings.Contains(image, "rhel-7-4-sap"):
+		t.Skip("disk expansion not supported on RHEL 7.4")
+	default:
```

Comment:
code execution halts at t.Skip, so no reason to capture it in a switch statement like this.

Reply:
Yeah that makes sense, fixed.